locale/sv/LC_MESSAGES/django.po

```diff
@@ -3822,3 +3822,6 @@ msgstr "Token existerar"
 
 msgid "Users permissions"
 msgstr "Användarens rättigheter"
+
+msgid "Other permitted build area (floor-m²)"
+msgstr "Annan bostadsbyggsrätt (k-m²)"
```

Comment:
Nice Swedish skills! :sweden:

Reply:
Thanks, I'm Finnswede from mothers side of the family 😄